index.Rmd

```diff
@@ -304,101 +303,127 @@ ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
+  geom_point() +
+  xlab("Language spoken most at home (number of Canadian residents)") +
+  ylab("Mother tongue (number of Canadian residents)") +
+  scale_x_log10(labels = scales::comma) +
+  scale_y_log10(labels = scales::comma)
+```
+</center>
+
+What do we see when considering the second part of our exploratory question? Do we see a difference in the pattern of the relationship between the number of people who speak a language as their mother tongue and the number of people who speak a language as their primary spoken language at home between higher-level language categories? Probably not!
+
+For each higher-level language category there appears to be a positive relationship between the number of people who speak a language as their mother tongue and the number of people who speak a language as their primary spoken language at home. This relationship looks similar, regardless of the category. 
+
+Does this mean that this relationship is positive for all languages in the world? Can we use this data visualization on its own to predict how many people have a given language as their mother tongue if we know how many people speak it as their primary language at home?
+
+The answer to both these questions is "no." However, with this exploratory data analysis, we can create new hypotheses, ideas,
 and questions (like the ones at the beginning of this paragraph). Answering those questions would likely involve gathering additional data and doing more complex analyses, which we will 
 see more of later in this course.  
 
 ### Putting it all together
 
-Below, we put everything from this chapter together in one code chunk. This demonstrates the power of R: in relatively few lines of code, we are able to create an entire data science workflow.
+Below, we put everything from this chapter together in one code chunk, with a few more layers to make the data visualization even more effective. This demonstrates the power of R: in relatively few lines of code, we are able to create an entire data science workflow.
 
-```{r nachos_to_cheesecake, fig.width=5.75, fig.height=4, warning=FALSE, message=FALSE}
+```{r nachos_to_cheesecake, fig.width=7.75, fig.height=4, warning=FALSE, message=FALSE}
 library(tidyverse)
 
-us_data <- read_csv("data/state_property_vote.csv")
-
-vote_data <- filter(us_data, party != "Not Applicable")
-
-ggplot(vote_data, aes(x = med_income, y = med_prop_val, color = party)) +
-  geom_point() +
-  xlab("Income (USD)") +
-  ylab("Median property value (USD)")
+can_lang <- read_csv("data/can_lang.csv")
+
+can_lang <- mutate(can_lang, mother_tongue = mother_tongue / 35151728)
+can_lang <- mutate(can_lang, most_at_home = most_at_home / 35151728)
+
+ggplot(can_lang, aes(x = most_at_home, 
+                     y = mother_tongue, 
+                     color = category,
+                     shape = category)) +
+  geom_point(alpha = 0.7) +
+  scale_color_manual(values = c("blue3","red3","black")) +
```

Comment:
For this last plot, I would colour it so it's the same as the one above since were "putting it all together" it'll show that its the same code as above a bit more

Reply:
OK, on Slack we discussed that I will keep the more effective plot, but in the writing above state that I add extra layers to do things, and then inline comment the extra new stuff. @leem44  agreed. This is reflected in 
b516099